#165020074 update travis CI to run tests

Karlmusingo opened the pull request:
#### What does this PR do?
Configure travisCI to run the tests
#### Description of Task to be completed?
~ write the configuration of travisCI in .travis.yml
#### How should this be manually tested?
Go to [travisCI ah-kgl-wakanda-frontend](https://travis-ci.com/andela/ah-kgl-wakanda-frontend)
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#165020074](https://www.pivotaltracker.com/story/show/165020074)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

Commit:
chore(.travis.yml): Update travisCI to run tests

configure travis to run the test

[Delivers #165020074]

kipropbrian reviewed (changes requested):
This PR should not have @gracelungu's commits, could you look into what the problem is?

gracelungu commented on .travis.yml:
```diff
@@ -0,0 +1,19 @@
+before_install:
+  - yarn install
+install:
+  - npm install
```
@Karlmusingo I would like to understand why you are using Yarn to install packages and NPM one after another! Aren't they suppose to do the same thing?

Karlmusingo replied:
We keep yarn

Karlmusingo commented:
> This PR should not have @gracelungu's commits, could you look into what the problem is?

Karlmusingo committed:
chore(.travis.yml): remove npm install

- Install dependecies on travis with yarn, not npm
- Add slack integration

[Delivers #165020074]

dusmel commented on .travis.yml:
```diff
@@ -0,0 +1,17 @@
+  - "stable"
+notifications:
+  email: false
+  slack: true
```
Hey @Karlmusingo from this line, does that mean we have integrated Travis with our slack channel?

Karlmusingo replied:
Yes @dusmel, I've just done it and you can see notifications in our integration channel.

kipropbrian commented:
The link to the PT board is not working. Could you link it [ #properly](https://pivotaltracker.com)
